src/loaders/zarr_utils/utils.ts

```diff
@@ -56,7 +73,9 @@ export function orderByTCZYX<T>(valsDimension: T[], orderTCZYX: TCZYX<number>, d
   orderTCZYX.forEach((val, idx) => {
     if (val >= 0) {
       if (val >= valsDimension.length) {
-        throw new Error("Unexpected axis index");
+        throw new VolumeLoadError(`Unexpected axis index in zarr: ${val}`, {
+          type: VolumeLoadErrorType.INVALID_METADATA,
+        });
```

Comment:
Love that this change also included being more descriptive about exactly what was wrong!